modules/sdp/SDPUtil.js

```diff
@@ -261,6 +261,29 @@ const SDPUtil = {
 
         return data;
     },
+
+    /**
+     * Gets the source name out of the name attribute "a=ssrc:254321 name:name1" and removes it from
+     * the {@code ssrcLines} array, so that it doesn't leak further up as source>parameter element in jingle.
+     *
+     * @param {string[]} ssrcLines
+     * @returns {string | undefined}
+     */
+    getAndRemoveSourceNameLine(ssrcLines) {
```

Comment:
I'm going to get rid of the removal side effect and actually filter out all parameters except for `msid` during the to jingle conversion